docs/core/config/auth/databases/ldap.md

```diff
@@ -356,75 +391,30 @@ The following variables can be used inside the `dovecot-ldap.conf.ext` files:
 | `%{ldap_multi:attrName:,}` | How to specify a comma `","` as separator, default is `""`. |
 | `%{ldap_multi:attrName:,:default}` | How to specify a comma `","` as separator, default explicitly defined. |
 | `%{ldap_dn}` | Retrieves the Distinguished Name of the entry. |
-| `%{ldap_ptr:attrName}` | Indirect fetch. Retrieves the attribute attrName, then it uses its content as a 2nd attrName where to fetch the actual value. |
-
-### Subqueries And Pointers
-
-LDAP values can now have DN pointers to other entries that are queried.
-
-::: info Note
-These aren't actually very useful anymore. See the next section for how
-to do multiple queries more easily using multiple userdbs.
-:::
-
-```
-user_attrs = \
-    =user=%{ldap:uid}, \
-    @mail=%{ldap:mailDN}, \
-    =uid=%{ldap:uidNumber@mail}, \
-    =gid=%{ldap:gidNumber@mail}, \
-    =home=%{ldap:rootPath@mail}/%d/%n
-```
-
-This will do a regular lookup first. Then does another lookup with DN taken
-from mailDN's value. The `@mail` attributes are assigned from the second
-lookup's results.
-
-```
-user_attrs = \
-    =user=%{ldap:uid}, \
-    =home=%{ldap_ptr:activePath}, \
-    !primaryPath, !secondaryPath
-```
-
-The activePath's value can be either `primaryPath` or `secondaryPath`.
-The home's value will be the contents of that field. The `!field` tells Dovecot
-to fetch the field's value but not to do anything with it otherwise.
 
 ### Multiple Queries via userdbs
 
 Example: Give the user a class attribute, which defines the default quota:
 
 ::: code-group
 ```[dovecot.conf]
-userdb {
+userdb db1 {
   driver = ldap
-  args = /etc/dovecot/dovecot-users-ldap.conf.ext
   result_success = continue-ok
+  fields {
+    class = %{ldap:userClass}
+    quota_rule = *:bytes=%{ldap:quotaBytes}
+  }
 }
 
-userdb {
+userdb db2 {
   driver = ldap
```

Comment:
userdb ldap {

Reply:
I can't easily see what is there, but if there are two userdb ldaps then yeah, I guess ldap1 ldap2 are ok..